### ISSUE TITLE
Comparison 

### ISSUE THREAD

**Issue by SajjadAemmi** [OmidSa75/fast_python_with_numba#1]
Hi, you can add "no numba version" of your demos, to we can better feel this comparison

**Comment by OmidSa75** [comment]
> Hi, you can add "no numba version" of your demos, to we can better feel this comparison

Yes. I will update this repository to compare different methods.
